modules/object/object_handler.go

```diff
@@ -126,9 +144,10 @@ func (m *Module) handleSealObject(ctx context.Context, block *tmctypes.ResultBlo
 		BucketName:           sealObject.BucketName,
 		ObjectName:           sealObject.ObjectName,
 		ObjectID:             common.BigToHash(sealObject.ObjectId.BigInt()),
-		OperatorAddress:      common.HexToAddress(sealObject.OperatorAddress),
+		Operator:             common.HexToAddress(sealObject.Operator),
 		SecondarySpAddresses: sealObject.SecondarySpAddresses,
 		Status:               sealObject.Status.String(),
+		SealedTxHash:         txHash,
```

Comment:
seems duplicate with updateTxHash

Reply:
we need to retain both sealedTxHash and updateTxHash as when an objected deleted, updateTxHash will be overwritten by this delete event